ynr/apps/twitterbot/management/commands/twitterbot_update_usernames.py

```diff
@@ -54,10 +55,21 @@ def handle_person(self, person):
 
                     if data:
                         if any(d["code"] == 63 for d in data["errors"]):
-                            return verbose(
+                            verbose(
```

Comment:
note to self: extract 44-79 into a new method and pass in a list of user ids and screen names

Reply:
@michaeljcollinsuk Should this be extracted to a helper method or a management method?

Reply:
@symroe ^^^

Reply:
Good question - I think this is best done in the management command, as there's already a lot of logic in here. I do generally like to keep management commands small, with logic in helpers, but in this case let's not start a refactor